README.md

```diff
@@ -196,11 +172,17 @@ cases. Detailed description of Ariadne configuration can be found in
 
 ## Usage [↑](#-ariadne)
 
-Launch `ariadne` executable to use TUI. If you [used
-Stack](#build-using-stack) to build Ariadne, type `stack exec --
-ariadne` to launch the executable. In order to use GUI launch
+Launch `ariadne` executable to use TUI. In order to use GUI launch
 `ariadne-qt`.
 
+If you used `nix-build` to build Ariadne you should have a bunch of
+`result/` folders. Find `ariadne` or `ariadne-qt` there. You can run
+`ariadne` directly. In order to use GUI version use `nix-shell
+shell-native.nix --run ariadne-qt`.
```

Comment:
`nix-shell shell-native.nix --run result/bin/ariadne-qt`, rather

Reply:
There is a way, but it would mean writing an inline Nix expression, which is not convenient on CLI.